### PR TITLE
Adds Necklaces for Gangsters

### PR DESCRIPTION
**Pull request by PKPenguin321**
Not available to sleeping carp.
Can be worn in the exosuit slot.
Offers extremely minor protection to energy and laser damage.
Has a 20 percent chance to reflect energy projectiles when worn.
Costs 10 influence.
Can be worn in any exosuit jacket's suit storage slot. This makes taking a jacket for your gang uniform have a benefit over taking a jumpsuit. (I'm pretty sure the reflect chance doesn't work when it's worn this way, however.)
Features codersprites by yours truly.
codersprites

This is my first PR, so if I accidentally nuke everything please be gentle :cry:

### CODE REVIEW

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -433,4 +433,26 @@
...
 
 /obj/item/clothing/head/winterhood/miner
-	icon_state = "winterhood_miner"
\ No newline at end of file
+	icon_state = "winterhood_miner"
+
+
+/obj/item/clothing/suit/necklace
+	name = "gold necklace"
+	desc = "Damn, it feels good to be a gangster."
```
Comment (Xhuis) [inline code/modules/clothing/suits/miscellaneous.dm]: pffffff

**Comment by kevinz000** [comment]
Won't this just make security's life harder? 20% reflect chance can mean that perfectly aimed taser shot be reflected and be a perfectly aimed taser shot at the shooter.

**Comment by PKPenguin321** [comment]
kevinz, that's the idea

**Comment by Lumipharon** [comment]
Pretty hilarious, but 20% chance for sec to cuck themselves is fairly strong.

**Comment by WJohn** [comment]
:-1: on the sprites, cute idea otherwise.

I'd love to see me some jewlery added to the game.
Also it'd probably be better if this was some kind of jumpsuit attachment, like ties are, rather than an exosuit item.

**Comment by PKPenguin321** [comment]
@WJohn If you have sprites that aren't codersprites, I'll add them in an instant.

And like I already said, they can work from the exosuit *storage* slot provided you're wearing a jacket (which gangs can take as their uniform). Any other slots could be stacked with things like HoS Armor + Gang Jumpsuit, and I don't really like the idea of that. HoS Armor + Gang Jumpsuit is already super strong and doesn't need a 20% chance to reflect.

**Comment by Zilenan91** [comment]
So do these fit in all exosuit slots? This means I can wear it in something like a Military Jacket out of the clothes vendor, right?

**Comment by PKPenguin321** [comment]
Only in exosuit slots of jackets, which the Military Jacket happens to be. It's like this because when gangs choose what uniform their gang should have, they have the option of jackets, which are generally not picked because they're inferior to jumpsuits.

**Comment by freerealestate** [comment]
the item sprite's not actually too bad as far as codersprites go
the on mob one looks a little weird due to the odd loop under the chin

**Comment by PKPenguin321** [comment]
:blush:

**Comment by Aloraydrel** [comment]
Thug life chose me

**Comment by GunHog** [comment]
If you REALLY wanted to go the extra mile, you could have the necklace match the gang's tag.

**Comment by Cheridan** [comment]
I don't really like this implementation. If you're adding a necklace, it should be an accessory item like medals and scarves. Having an accessory-like item that isn't really an accessory is inconsistent.

**Comment by optimumtact** [comment]
https://github.com/optimumtact/-tg-station/tree/necklaces rescued your branch for you

**Comment by Iamgoofball** [comment]
Why don't we add a neck slot

**Comment by MrStonedOne** [comment]
Please make a changelog,

You can add it to the PR's body rather then as a file now, see here: https://tgstation13.org/phpBB/viewtopic.php?f=5&t=5157

**Comment by PKPenguin321** [comment]
eh, fuck it, i honestly dont care about this that much anymore and cant be bothered to fix conflicts and add a changelog

maybe this will come back as a cosmetic or something later. rip tho

**Comment by Iamgoofball** [comment]
>closing the most swag pr

fuck no

**Comment by PKPenguin321** [comment]
@Iamgoofball feel free to steal the shit out of my code/codersprites and do something with them if you want

**Pull request closed without merging**